scene/resources/material.h

```diff
@@ -305,6 +317,33 @@ class BaseMaterial3D : public Material {
 		DISTANCE_FADE_MAX
 	};
 
+	enum StencilMode {
+		STENCIL_MODE_DISABLED,
+		STENCIL_MODE_OUTLINE,
+		STENCIL_MODE_XRAY,
+		STENCIL_MODE_CUSTOM,
+		STENCIL_MODE_MAX // not an actual mode, just the amount of modes
```

Comment:
```suggestion
		STENCIL_MODE_MAX // Not an actual mode, just the amount of modes.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).